src/Colourista/IO.hs

```diff
@@ -47,6 +48,11 @@ import qualified Colourista.Pure as Colourista
 -- Direct IO functions
 ----------------------------------------------------------------------------
 
+-- | Print 'Text' coloured in specified RGB notaion
+rgbMessage :: HasColourMode => Word8 -> Word8 -> Word8 -> Text -> IO ()
```

Comment:
We have removed colouring mode support in the `main` branch in `colourista`. Could you please rebase and remove left-overs of `HasColourMode`? 🙂